.github/workflows/trigger-nightly.yml

```diff
@@ -0,0 +1,46 @@
+name: Trigger nightly build
+
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '0 * * * *'
+  workflow_dispatch:
+
+jobs:
+  trigger-nightly:
+    name: Push tag for nightly build
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: 'Checkout'
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.NIGHTLY_BUILD_GH_TOKEN }}
+      -
+        name: 'Push new tag'
+        # TODO determine nightly version based on latest minor version
+        run: |
+          git config user.name "${GITHUB_ACTOR}"
+          git config user.email "${GITHUB_ACTOR}@users.noreply.github.com"
+
+          TAG=v0.2.0-nightly.$(date +'%s')
+          git tag -a $TAG -m "$TAG: nightly build"
+          git push origin $TAG
+      - name: 'Clean up nightly releases'
```

Comment:
What if we didn't commit anything for 7 days? We will be left without a nightly release, right? 😕  I would rather make sure we always have at least 1 nightly release - is there an easy way to configure this (i.e. we don't delete it if it's the last one)?